test/support/test_config.h

```diff
@@ -209,7 +209,7 @@
 // within a sycl kernel which MSVC uses to allow comparisons with literal zero without warning
 #define _PSTL_TEST_COMPARISON_BROKEN                                                                                   \
     ((__cplusplus >= 202002L || _MSVC_LANG >= 202002L) && _MSVC_STL_VERSION >= 143 && _MSVC_STL_UPDATE >= 202303L &&   \
-    __INTEL_LLVM_COMPILER > 0 && __INTEL_LLVM_COMPILER <= _PSTL_TEST_LATEST_INTEL_LLVM_COMPILER)
+    __INTEL_LLVM_COMPILER > 0 && __INTEL_LLVM_COMPILER <= 20250000)

```

Comment:
Should all of these be `20250004` since we had some patch releases? I have not checked, but I would assume the same issues probably apply to the patch release.

Reply:
what about `< 20250100`

Reply:
> what about `< 20250100`

That sounds better to me than my suggestion.

Reply:
I also have thinks about this variant and prefer it.

Reply:
Agree, done